data/palettes.json

```diff
@@ -6,28 +6,53 @@
         "buttonsRows": [
             [
                 {
-                    "title": "Cross out (⌨: shift+delete)",
+                    "id": "crossOutBtn",
+                    "title": "Cross out",
                     "additionalOnclick": "MathLivePasteFromButton(this); GoogleAnalytics('Cross Out');",
-                    "value": "$$\\enclose{updiagonalstrike downdiagonalstrike}[2px solid red]{\\blacksquare}$$"
+                    "value": "$$\\enclose{updiagonalstrike downdiagonalstrike}[2px solid red]{\\blacksquare}$$",
```

Comment:
red -> blue or delete, this is overriden in css however it would be better to make it consistent